test-cases/performance/perf-regression.100threads.30M-keys-ebs-volumes.yaml

```diff
@@ -0,0 +1,28 @@
+test_duration: 200
+
+stress_cmd_w: "cassandra-stress write no-warmup cl=QUORUM duration=60m -schema 'replication(factor=3)' -port jmx=6868 -mode cql3 native -rate threads=25 -pop seq=1..30000000"
```

Comment:
Why here the pop is seq and for the read and mixed it's gauss?

Reply:
It is the same as in base conf file: perf-regression.100threads.30M-keys.yaml